pytest-slc/README.md

```diff
@@ -14,15 +14,28 @@ pip install pytest-exasol-slc
 
 ## Usage in Tests
 
-Below is an example of a test that requires a script language container to be uploaded into the database.
+Below is an example of a test that requires a script language container to be built and uploaded into the database.
 Note, that by default this test will run twice - once for each backend.
 
 ```python
-import pyexasol
-
-def test_something_with_slc(upload_slc, container_file_path, language_alias):
-
-    upload_slc(container_file_path=container_file_path, language_alias=language_alias)
-
-    # Now run the actual test
+import pytest
+
+@pytest.fixture(scope='session', autouse=True)
+def extension_export_slc_async(export_slc_async, use_onprem, use_saas):
+    if use_onprem or use_saas:
```

Comment:
maybe also add an explicit option to activate it, in case it needed in a special case also without a backend, maybe to inspect the tar file